client-src/elements/chromedash-footer_test.ts

```diff
@@ -9,7 +9,7 @@ describe('chromedash-footer', () => {
     );
     assert.exists(component);
     assert.instanceOf(component, ChromedashFooter);
-    const footer = component.shadowRoot.querySelector('footer');
+    const footer = component.shadowRoot!.querySelector('footer');
```

Comment:
See if this works to avoid the `!`:
```suggestion
    const footer = component.renderRoot.querySelector('footer');
```